docs/projects/tnp-website/frontend/components/testimonials.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Testimonials
+id: testimonials
+description: Testimonials Component in T&P Website
+sidebar_label: Testimonials
+keywords: [astrojs, open-source, docs]
+---
+Introducing Testimonials of TNP Website!
+
+The **Student Testimonials** and **Company Testimonials** section of the website provides you feedbacks and experiences shared by the students and the companies visiting in TCET. 
+It is a responsive and touch-enabled slider which includes data like feedback, student's image or company logo, name and position. It supports slide touch gestures and is situated at the bottom of the [Home Page](https://tnp.tcetmumbai.in/) of TNP website. 
+
+### Adding Testimonials
+
+To add reviews on the website, we need to edit `studentTestimonials` or `companyTestimonial` array in `index.astro` file. Both of these array contain multiple testimonial objects of the type `TestimonialType`.
+
+Here's how we made those changes: 
+
+1. Opening `index.astro` file in our projects root directory
+
+2. Locating the  `studentTestimonials` or `companyTestimonial` array
+
+3. To add a new testimonial, we created a new object which would be defined as follows:
+
+   ```jsx title="index.astro"
+       {
```

Comment:
It would be great if you can include Line Numbers in the code sections.
Also, highlight the part which you are talking about.